apps/designer/app/designer/designer.tsx

```diff
@@ -67,11 +73,18 @@ const useSubscribeSyncStatus = () => {
   useSubscribe<"syncStatus", SyncStatus>("syncStatus", setValue);
 };
 
+const useNavigatorLayout = () => {
+  // We need to render the detached state only once the setting was actually loaded from local storage.
+  // Otherwise we may show the detached state because its the default and then hide it immediately.
```

Comment:
I think if we'd store settings in cookies instead of local storage this would be solvable as cookies available at the server as well.

Also, maybe we should consider not doing SSR at all for the editor? Usually such highly dynamic UIs don't benefit much from SSR.

Reply:
I was considering going over cookie, but then you loose the setting on logout, which is not quite what we need here. We assume people will choose the setting for the screen and not change it for a long time. 

I am seriously considering to stop doing majority of SSR for the designer and only render a more static part of it. For now this has been solved though.

Reply:
Each setting could be a separate cookie which we wouldn't delete on logout. But yeah, removing SSR sounds like a better solution